data-source/jsonnet/common/blocks/individual/employment/main_employment_block.jsonnet

```diff
@@ -7,51 +7,44 @@ local rules = import '../../../lib/rules.libsonnet';
   content_variants: [
     {
       content: {
+
         title: 'Main job',
-        contents: [
-          {
-            description: 'The next set of questions is about your main job. Your main job is the job in which you usually work the most hours',
-          },
-        ],
+        contents: [{ description: 'The next set of questions is about your main job. Your main job is the job in which you usually work the most hours.' }],
```

Comment:
This formatting style (and in the rest of the diff) is inconsistent with how we're indenting content/braces in other js/jsonnet files.

Reply:
Its kind of annoying that the jsonnet fmt command doesn't sort this out...

Reply:
Well that's why I left it like that, because jsonnet's own linter did it.

Reply:
Jsonnet's formatter is weird, it will format based on your initial structure. So if you broke the start of the object to a new line, then it will format the rest accordingly. It's not ideal.